### PR TITLE
Chore: Update READMEs

### PR DESCRIPTION
**Pull request by promaty**
Updated missing readmes for server, services and shared

### CODE REVIEW

File: packages/services/README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ docker-compose up -d
 
 It provides the following list of worker services:
 - [`Heartbeat`](./src/workers/heartbeat.js): It will try to update the court in case it is outdated. Configured initially to run one time per hour indefinitely.
+- [`Monitor Keeper`](./src/workers/monitor-keeper.js): ADD SOMETHING
```
Comment (promaty) [inline packages/services/README.md:18]: here

File: packages/shared/README.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ The first one is only used by the back-office frontend app where all the compone
 ### Helpers
 
 It provides the following helper functions:
+- [`email-client`](./helpers/email-client.js): Handles sending emails through Postmark
+- [`gas-price-oracle`](./helpers/gas-price-oracle.js): ADD SOMETHING
+- [`get-wallet-from-pk`](./helpers/get-wallet-from-pk.js): ADD SOMETHING
```
Comment (promaty) [inline packages/shared/README.md:31]: here

File: packages/shared/README.md [hunk 3]
```diff
@@ -26,6 +26,11 @@ The first one is only used by the back-office frontend app where all the compone
...
 - [`logger`](./helpers/logger.js): Logger object that provides a friendly interface for fancy logging 
 - [`numbers`](./helpers/numbers.js): BigNumber-related helper functions
 - [`sleep`](./helpers/sleep.js): Sleep function to wait a number of seconds 
+- [`times`](./helpers/times.js): Time constants for using with `Date()`
+- [`voting`](./helpers/voting.js): ADD SOMETHING
```
Comment (promaty) [inline packages/shared/README.md:36]: and here

File: packages/server/README.md [hunk 4]
```diff
@@ -17,72 +17,379 @@ All the provided endpoints are `Content-Type: application/json`
 
 #### 1. Users
 
-##### 1.1. Create
+##### 1.1. Check existing subscription details
```
Comment (facuspagnuolo) [inline packages/server/README.md:20]: I think we should list these separately: create and get

File: packages/server/README.md [hunk 5]
```diff
@@ -17,72 +17,379 @@ All the provided endpoints are `Content-Type: application/json`
...
+  - Code: `200 OK`
+  - Body:
+
+  ```
+  {
+    "email": "new-juror@aragoncourt.com" || null
+  }
+  ```
+
+##### 1.6. Subscribe juror / Change juror email
```
Comment (facuspagnuolo) [inline packages/server/README.md:144]: I think the subscription part is not documented below

File: packages/services/README.md (hunk 1 above)
Comment (facuspagnuolo) [inline packages/services/README.md:18]:
```suggestion
- [`Monitor Keeper`](./src/workers/monitor-keeper.js): Loops over all the transactions sent from the keeper address looking for suspicious transactions to report them.
```

File: packages/shared/README.md (hunk 2 above)
Comment (facuspagnuolo) [inline packages/shared/README.md:31]:
```suggestion
- [`get-wallet-from-pk`](./helpers/get-wallet-from-pk.js): Decode Ethereum address based on a private key
```

File: packages/shared/README.md [hunk 6]
```diff
@@ -26,6 +26,11 @@ The first one is only used by the back-office frontend app where all the compone
 ### Helpers
 
 It provides the following helper functions:
+- [`email-client`](./helpers/email-client.js): Handles sending emails through Postmark
+- [`gas-price-oracle`](./helpers/gas-price-oracle.js): ADD SOMETHING
```
Comment (facuspagnuolo) [inline packages/shared/README.md:30]:
```suggestion
- [`gas-price-oracle`](./helpers/gas-price-oracle.js): Get gas price oracle object used to know the current gas prices being paid on each network
```

File: packages/shared/README.md (hunk 3 above)
Comment (facuspagnuolo) [inline packages/shared/README.md:36]:
```suggestion
- [`voting`](./helpers/voting.js): Utils related to the CR Voting module of Aragon Court
```

File: packages/server/README.md (hunk 4 above)
Comment (promaty) [reply packages/server/README.md:20]: Ah yes I removed the create endpoint that is used from anj.
I will move Create endpoint to the end and name it "Create user without verifying"

File: packages/shared/README.md (hunk 3 above)
Comment (promaty) [reply packages/shared/README.md:36]: What is CR?

File: packages/server/README.md (hunk 5 above)
Comment (promaty) [reply packages/server/README.md:144]: It's the same PUT request, I just copied this from notion

File: packages/server/README.md (hunk 4 above)
Comment (promaty) [reply packages/server/README.md:20]: added at 1.11

**Pull request merged**